airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -4,25 +4,38 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Github Source Spec",
     "type": "object",
-    "required": ["access_token", "repository", "start_date"],
+    "required": ["access_token", "start_date"],
     "additionalProperties": false,
     "properties": {
       "access_token": {
         "type": "string",
         "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
         "airbyte_secret": true
       },
+      "organization": {
+        "type": "string",
+        "examples": ["airbytehq/airbyte"],
+        "description": "Space-delimited list of GitHub organizations. e.g. `airbytehq singer-io`"
+      },
       "repository": {
         "type": "string",
         "examples": ["airbytehq/airbyte"],
-        "description": "GitHub repository. e.g. 'airbytehq/airbyte' or 'singer-io/tap-github'"
+        "description": "Space-delimited list of GitHub repositories, e.g. `airbytehq/airbyte singer-io/tap-github`"
```

Comment:
this should be a string list. The UI supports lists and it looks cleaner that way. We should have a clean story for backwards compatibility though.